source/vstd/arithmetic/power2.rs

```diff
@@ -133,33 +158,6 @@ pub proof fn lemma_pow2_strictly_increases_auto()
     }
 }
 
-/// Proof that, for the given positive number `e`, `(2^e - 1) / 2 == 2^(e - 1) - 1`
-pub proof fn lemma_pow2_mask_div2(e: nat)
```

Comment:
I checked that this is not used in the `verus` repository itself, but it just occurred to me that it might be by an external verus user. How large is the verus ecosystem at this point, can we check all/most of the projects using it? What sort of stability guarantees does verus offer at this point, if any?

My thought was that if we knew it's unused, the same lemma might be more naturally stated in terms of the `mask(n)` spec. See `lemma_mask_div2{,_auto}` below.

Reply:
> How large is the verus ecosystem at this point, can we check all/most of the projects using it?

We don't yet have a system for that, and even when it's up and running, it will only have partial coverage.

> What sort of stability guarantees does verus offer at this point, if any?

No official guarantees yet.